### DISCUSSION TITLE
Support player glow color for chat message formatting

### DISCUSSION THREAD

**Discussion by Marotheit** [Zrips/CMI#8196] [Enhancement]
Player glow color seems to be supported in the message format, but not the chat format.

I was hoping to allow players to change their name color in chat, but wanted to keep their message color the same for readability.

**Comment by Zrips** [comment]
You can use either %cmi_user_chatcolor% placeholder if player picked color with /cmi chatcolor command or %cmi_user_glow_code% if its for glow color and just insert this into chat message format before their name. Tho keep in mind that display name could override this, so you might even want to modify that one by adding appropriate placeholder

**Reply by Marotheit** [reply]
I'll give that a try when I get home. I don't allow players to modify their nicknames so hopefully that won't be an issue.

I didn't think to try %cmi_user_glow_code%. I wasn't sure if placeholders worked in that situation besides the ones listed above ({world}, etc.)
